.gitmodules

```diff
@@ -3,4 +3,4 @@
 	url = https://github.com/arfc/squirrel
 [submodule "moose"]
 	path = moose
-	url = ../../idaholab/moose
+	url = https://github.com/idaholab/moose
```

Comment:
Why this change? This is here so that people who clone zapdos with ssh would do the same for submodules

Reply:
I have always been confused about this convention, as I thought it was just for convenience. Because of that, I always preferred typing the URL so I could click through to the appropriate code for the submodule if I needed to look something up online. 

If this is for SSH, why does only MOOSE use a different convention and not Squirrel?

Reply:
Because I didn't know about the relative path usage when I added the squirrel submodule and for some reason I was too lazy to change one other line when I added the moose submodule. I'd like to have this changed back, and maybe if you're less lazy than I am you can change the squirrel url 😃

Reply:
Will do!